packages/resin-universal-ga/src/browser.js

```diff
@@ -13,16 +15,22 @@ module.exports = function (propertyId, site, debug) {
 				options.cookieDomain = 'none'
 			}
 			window.ga('create', propertyId, site, TRACKER_NAME, options)
+			loggedIn = true;
 		},
 		logout: function () {
+			if (!loggedIn) throw new Error("Can't record GA logout event before login");
```

Comment:
This doesn't log anything. Also the clearer fix would be here https://github.com/resin-io-modules/resin-analytics/blob/master/packages/resin-event-log/src/resin-event-log.js#L73 to not call the adaptors' logout method if the userId is null-ish